### PR TITLE
fix build image builds temporarily

### PR DESCRIPTION
**Pull request by arroyc**
please take a look, temporarily disabled apt-get upgrades to make the build work

### CODE REVIEW

File: images/build/Dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@ ENV LANG C.UTF-8
 
 # Install basic build tools
 RUN apt-get update \
-    && apt-get upgrade -y \
+#    && apt-get upgrade -y \
```
Comment (kichalla) [inline images/build/Dockerfile]: Why these changes?

Comment (kichalla) [reply images/build/Dockerfile]: just FYI...we added the upgrade to avoid getting flagged by S360 for missing security updates...

**Comment by arroyc** [comment]
closing on behalf of #486
